tests/test_querysets.py

```diff
@@ -0,0 +1,139 @@
+from .base import BaseTinyFatTest
+from .utils import match_all_elements
+from tinyfatdb import TinyFatDB
+from tinydb import Query as Q
+
+
+########################################################################
+class TestTinyFatQueryset(BaseTinyFatTest):
+
+    ####################################################################
+    def setUp(self):
+        self.db = TinyFatDB.create()
+        self.entries = ({"a": "A1", "b": "B1", "c": "C1"},
+                        {"a": "A2", "b": "B2", "c": "C2"},
+                        {"a": "A3", "b": "B3", "c": "C3"})
+        eids = self.db.insert_multiple(self.entries)
+        for entry in self.entries:
+            entry.update({"eid": eids.pop(0)})
```

Comment:
I have not looked into this, but my gut tells me that this may not be the best way to go about gathering the `eid`s for each entry. If something got mixed up with the ordering, then you would have at least two entry information with the wrong `eid`.

This method will work fine if the list returned from `insert_multiple` and the tuple of entries will always have the exact same ordering. I am not sure if that is the case.